app/src/main/java/com/google/android/soundchecker/utils/ui/WaveformDisplay.kt

```diff
@@ -12,12 +28,20 @@ import androidx.compose.ui.geometry.Offset
 import androidx.compose.ui.graphics.*
 import androidx.compose.ui.unit.dp
 
+/**
+ * Compostable that displays a waveform.
```

Comment:
I'm worried that this Compostable code may be prone to bit rot.

Reply:
I agree. Now that there is no separate section for the UI like there was for xml, we should be diligent whenever it makes sense to refactor code.

Reply:
I should not have tried to be funny. Note 't'
Compostable vs Composable